exporters/jaeger/src/test/java/io/opentelemetry/exporter/jaeger/JaegerGrpcSpanExporterTest.java

```diff
@@ -95,7 +95,7 @@ void testExport() throws Exception {
             .setName("GET /api/endpoint")
             .setStartEpochNanos(TimeUnit.MILLISECONDS.toNanos(startMs))
             .setEndEpochNanos(TimeUnit.MILLISECONDS.toNanos(endMs))
-            .setStatus(Status.ok())
+            .setStatus(StatusData.ok())
```

Comment:
do we need a StatusData? Would the API's StatusCode work for this?

Reply:
never mind. we need the description.

Reply:
`StatusData` is `StatusCode` + (`String`) description